lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -323,6 +323,26 @@ struct ListCreateOpConversion
   }
 };
 
+struct MapCreateOpConversion : public OpConversionPattern<firrtl::MapCreateOp> {
+  using OpConversionPattern::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(firrtl::MapCreateOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    auto mapType = getTypeConverter()->convertType<om::MapType>(op.getType());
+    if (!mapType)
+      return failure();
+    auto keys = adaptor.getKeys();
+    auto values = adaptor.getValues();
+    SmallVector<Value> tuples;
+    for (auto [key, value] : llvm::zip(keys, values))
+      tuples.push_back(rewriter.create<om::TupleCreateOp>(
+          op.getLoc(), ArrayRef<Value>{key, value}));
```

Comment:
Could this be simply `{key, value}`? Not important, and if you prefer this regardless works for me.

Reply:
Tried  `{key, value}` but didn't work without explicit types 😞

Reply:
Okay, thanks!